From 966aae39fb309ed56c3836926623c5ce858ce077 Mon Sep 17 00:00:00 2001
From: Arian <arian.kulmer@web.de>
Date: Wed, 15 Mar 2023 11:15:25 +0100
Subject: [PATCH] SessionConfigurationUtils: Allow privileged camera apps to create raw streams for raw capable cameras

Change-Id: I4e82027917e458e1472464e7317bb1968ff7ee2c
---

diff --git a/services/camera/libcameraservice/utils/SessionConfigurationUtils.cpp b/services/camera/libcameraservice/utils/SessionConfigurationUtils.cpp
index eb45fbe..7e80052 100644
--- a/services/camera/libcameraservice/utils/SessionConfigurationUtils.cpp
+++ b/services/camera/libcameraservice/utils/SessionConfigurationUtils.cpp
@@ -198,7 +198,8 @@
         auto entry = info.find(ANDROID_REQUEST_AVAILABLE_CAPABILITIES);
         for (size_t i = 0; i < entry.count; ++i) {
             uint8_t capability = entry.data.u8[i];
-            if (capability == ANDROID_REQUEST_AVAILABLE_CAPABILITIES_LOGICAL_MULTI_CAMERA) {
+            if (capability == ANDROID_REQUEST_AVAILABLE_CAPABILITIES_LOGICAL_MULTI_CAMERA ||
+                    capability == ANDROID_REQUEST_AVAILABLE_CAPABILITIES_RAW) {
                 isLogicalCamera = true;
                 break;
             }
